nornir/plugins/inventory/netbox.py

```diff
@@ -95,10 +88,45 @@ def __init__(
                 host["data"]["model"] = d["device_type"]
                 host["platform"] = d["platform"]
 
+            # Add NAPALM connection options if they exist for the host platform
+            if napalm_args.get(host["platform"]):
+                host["connection_options"]["napalm"] = {
+                    "extras": {
+                        "optional_args": napalm_args.get(host["platform"], {})
+                        if d["platform"]
```

Comment:
Is this if/else clause needed? Isn't this redundant with the .get() call on napalm_args?